maintenance/adal-node/package.json

```diff
@@ -26,26 +26,26 @@
     "node": ">= 0.6.15"
   },
   "dependencies": {
-    "@types/node": "^8.0.47",
+    "@xmldom/xmldom": "^0.7.0",
     "async": "^2.6.3",
+    "axios": "^0.21.1",
```

Comment:
The other updates here are related dependencies?

Reply:
A couple things happened here:

- `xmldom` replaced with `@xmldom/xmldom`
- Other unrelated dependecies reordered to be alphabetical (npm cli does this when you install something)
- `@types/node` package updated to newer version (provides types for built-in Node APIs)
- `typescript` (used for tests) updated to v3 to be compatible with new Node types.

(tests werent passing until I did the last two)